models/enums_and_bitflags.py

```diff
@@ -320,7 +320,7 @@ def string_to_bitflag(value, dictionary):
             original_keyword = keyword
             keyword = EnumConverter.c(keyword)
             if keyword not in dictionary:
-                raise ValueError(f'Keyword "{original_keyword}" not recognized in dictionary')
+                raise ValueError(f'Keyword "{original_keyword.strip()}" not recognized in dictionary')
```

Comment:
Yeah that's what I was thinking but we also remove all spaces (leading or otherwise) in the `convert` method. So this is actually going to lead you down the wrong path if you leave the spaces (as it did for me).

Reply:
I don't see the `.strip()` in `convert`.  (I looked for it before making this comment.)  What am I missing?

Reply:
There is a `replace(' ', '')` command in the `c` function.